hack/lib/build/binaries.sh

```diff
@@ -342,14 +354,10 @@ function os::build::place_bins() {
       done
 
       # Create binary copies where specified.
-      local suffix=""
-      if [[ $platform == "windows/amd64" ]]; then
-        suffix=".exe"
-      fi
       for linkname in "${OC_BINARY_COPY[@]}"; do
         local src="${release_binpath}/oc${suffix}"
         if [[ -f "${src}" ]]; then
-          ln "${release_binpath}/oc${suffix}" "${release_binpath}/${linkname}${suffix}"
+          cp "${release_binpath}/oc${suffix}" "${release_binpath}/${linkname}${suffix}"
```

Comment:
does this need to be cp -f to overwrite the link?

Reply:
sure, fixed